tutorials/abap-environment-behavior-action/abap-environment-behavior-action.md

```diff
@@ -760,15 +747,13 @@ In this tutorial, wherever XXX appears, use a number (e.g. 000).
       ![Enhance behavior definition for projection view](projection5.png)
 
       **Hint: If your UI buttons don't appear, please add a space anywhere you want in your data definition `ZI_TRAVEL_M_XXX` and behavior definition `ZI_TRAVEL_M_XXX`, activate both and open your SAP Fiori preview again. It takes 120 seconds to make the buttons visible on the user interface.**
```

Comment:
This should be formatted as a note.

Reply:
Merve, did you already incorporate this suggestion regarding the note?  I did not see it in the changed files and wanted to be sure before I merged it.

Reply:
Yes, I formatted the hint section.